server/myenv/Lib/site-packages/django/apps/registry.py

```
@@ -0,0 +1,437 @@
+            self.ready = True
+            self.ready_event.set()
+
+    def check_apps_ready(self):
+        """Raise an exception if all apps haven't been imported yet."""
+        if not self.apps_ready:
+            from django.conf import settings
+
+            # If "not ready" is due to unconfigured settings, accessing
+            # INSTALLED_APPS raises a more helpful ImproperlyConfigured
+            # exception.
+            settings.INSTALLED_APPS
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)